documentation.md

```diff
@@ -97,22 +97,24 @@ Your container's selector
 
 The purchase amount (in euro cents)
 
-- plans: `Plan[]` [optional]
+- plans: `Plan[]` [optional] : [more info below](#plan-option-plan)
 
 An array of the plans you want to display. If not provided, the widget returns all your available payment plans.
 
-- locale: `fr|en|es|it|de|nl` [optional, default: en]
+- locale: `fr|en|es|it|de|nl|pt` [optional, default: en]
 
 - clickableSelector: `string` [optional, default: null]
 
 If provided, the modal will open when the element matching this query selector is clicked.
 
-- cards: ('cb' | 'visa' | 'amex' | 'mastercard')[] [optional]
+- cards: `cb|visa|amex|mastercard`[] [optional]
 
 Display card logos in the modal
 
 ## Plan option: `Plan`
 
+You can customize the displayed plans with this parameter. You can hide a plan that would be displayed otherwise by adding the other plans with those information:
```

Comment:
```suggestion
You can customize the displayed plans with this parameter. You can hide a plan that would be displayed otherwise by adding the other plans with this information:
```
`Information` has no plural form and is uncountable : )

Reply:
Thanks